01-introduction.Rmd

```diff
@@ -89,44 +85,62 @@ It can be seen as a part of Geography\index{geography}, which has a 2000+ year h
 
-This 'geodata revolution' drives demand for high performance computer hardware and efficient, scalable software to handle and extract signal from the noise, to understand and perhaps change the world.
-Spatial databases\index{spatial database} enable storage and generation of manageable subsets from the vast geographic data stores, making interfaces for gaining knowledge from them vital tools for the future.
-R is one such tool, with advanced analysis, modeling and visualization capabilities.
-In this context the focus of the book is not on the language itself [see @wickham_advanced_2019].
-Instead we use R as a 'tool for the trade' for understanding the world, similar to Humboldt's\index{von Humboldt} use of tools to gain a deep understanding of nature in all its complexity and interconnections [see @wulf_invention_2015].
-Although programming can seem like a reductionist activity, the aim is to teach geocomputation\index{geocomputation} with R not only for fun, but for understanding the world.
+Early geographers used a variety of tools including barometers, compasses and [sextants](https://en.wikipedia.org/wiki/Sextant) to advance knowledge about the world [@wulf_invention_2015]. 
+It was only with the invention of the marine [chronometer](https://en.wikipedia.org/wiki/Marine_chronometer) in 1761 that it became possible to calculate longitude at sea, enabling ships to take more direct routes, for example.
+Before the turn of the century, there was an accute shortage of data and tools for geographic analysis.
+<!-- Using a sextant was a difficult-to-learn but essential tool for navigation in the 18th century.
+Likewise, learning to program and write and share reproducible code can be hard, but is a key skill for modern geocomputation. -->
+<!-- Maps were predominantly hand-drawn until modern computing enabled digitisation, a process which only become widespread in the 1990s and which is still ongoing [@gold_outsidein_1996; @auffret_histmapr_2017]. -->
+
+Nowadays, researchers and practitioners have no such limitations and in some cases face the opposite problem: too much data; too many tools.
+Most phones now have a global positioning (GPS\index{GPS}) receiver.
+Sensors ranging from satellites and semi-autonomous vehicles to citizen scientists incessantly measure every part of the world.
+The rate of data produced can be overwhelming, with emerging technologies such as autonomous vehicles generating hundreds or even thousands of GB of data daily.
+Remote sensing\index{remote sensing} datasets from satellites are too large to analyze with a single computer, as outlined in Chapter \@ref(gis).
+This 'geodata revolution' drives demand for high performance computer hardware and efficient, scalable software to handle and extract signal from the noise.
+Evolving open source tools can import and process subsets from the vast geographic data stores directly, and via APIs and via interfaces to databases. \index{spatial database}
```

Comment:
Something is missing in this sentence. Do you mean that we can use APIs for an easy cli-driven data access and other interfaces for interacting with databases (import, manipulation and export again)?

Reply:
Will look this eve, thanks for the review!